docs/nima/pico/pico_intro.adoc

```diff
@@ -101,16 +175,26 @@ The Pico API provides types that are generally useful at compile time to assign
 
 == Configuration
 
-Pico Config builder
+Configuration can be achieved through a Pico add-on that is based on Helidon's configuration subsystem, and adds support for something called _config-driven services_ using the `@ConfiguredBy` annotation. 
 
+When applied to a target service interface, the `@ConfiguredBy` annotation will allow you to use a higher level aggregation for application configuration, and then allow the configuration to drive activation of services in the Pico Framework.
 
+NOTE: You can use Pico configuration in combination with the Helidon configuration subsystem. 
 
-== Examples
+There are a few additional caveats to understand about `ConfiguredBy` and its supporting infrastructure.
 
-//optional 
+* `@ConfigBean Builder` is used to aggregate configuration attributes to this higher-level, application-centric configuration beans.
```

Comment:
right, this refers us back to the ConfigBean builder section - ideally with a link.